catroid/src/main/res/values/strings.xml

```diff
@@ -1457,7 +1448,6 @@ device\'s geolocation sensor, and in order to import and export projects from an
 needs read and write access to it. You can always change permissions through your device\'s settings.</string>
 
     <!-- Error messages -->
-    <string formatted="false" name="error_project_upload">Something went wrong while uploading the project.</string>
```

Comment:
Please remove this also in all the translations. Contact me via Slack, and I can show you how to do it fast.